centaur/src/main/resources/standardTestCases/cwl_relative_imports_url.test

```diff
@@ -4,6 +4,8 @@ workflowType: CWL
 workflowTypeVersion: v1.0
 workflowRoot: relative_imports
 skipDescribeEndpointValidation: true
+# This test has experienced sporadic failures, and CWL is not fully supported. Therefore the test is being disabled.
+ignore: true
```

Comment:
Evidence from log that this is being ignored00

[33m- should successfully run cwl_relative_imports_url !!! IGNORED !!![0m